src/components/Button.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import styled from 'styled-components'
+import buttonBG from '../assets/hc_button.svg'
+import { hexToRgba } from '../utility/utilities'
+
+const buttonSize = {
+  small: '60px',
+  default: '80px',
+  large: '300px',
+}
+
+const buttonColors = ['primary', 'secondary', 'tertiary']
+
+const StyledButton = styled.button`
+  display: block;
+  text-decoration: none;
+  font-family: 'HK Grotesk';
+  font-weight: ${p => p.theme.typography.h2.weight};
+  border: transparent;
+  transition: all 250ms;
+  min-width: ${p => (p.size ? buttonSize[p.size] : buttonSize['default'])};
+  text-align: center;
+  padding: 0.75em;
+  border-radius: 3px;
+  margin: 1em;
+  :hover {
+    cursor: pointer;
+  }
+  :focus {
```

Comment:
I'd almost want to remove this - because if we add new variants we have to remember to include a focus, which we've already forgotten for the tertiary variant (no visible change on focus to tertiary button, not good for keyboard accessibility)

Reply:
focus has been added! it's different than designs so i'll be touching base w. them